code/datums/supplypacks/robotics.dm

```diff
@@ -0,0 +1,103 @@
+/*
+*	Here is where any supply packs
+*	related to robotics tasks live.
+*/
+
+/datum/supply_packs/robotics
+	group = "Robotics"
+
+/datum/supply_packs/randomised/robotics
+	group = "Robotics"
+	access = access_robotics
+
+/datum/supply_packs/eng/robotics
+	name = "Robotics assembly crate"
+	contains = list(
+			/obj/item/device/assembly/prox_sensor = 3,
+			/obj/item/weapon/storage/toolbox/electrical,
+			/obj/item/device/flash = 4,
+			/obj/item/weapon/cell/high = 2
+			)
+	cost = 10
+	containertype = /obj/structure/closet/crate/secure/gear
+	containername = "Robotics assembly"
+	access = access_robotics
+
+/datum/supply_packs/robotics/robolimbs_basic
+	name = "Basic robolimb blueprints"
+	contains = list(
+			/obj/item/weapon/disk/limb/morpheus,
+			/obj/item/weapon/disk/limb/xion
+			)
+	cost = 15
+	containertype = /obj/structure/closet/crate/secure/gear
+	containername = "Robolimb blueprints (basic)"
+	access = access_robotics
+
+/datum/supply_packs/robotics/robolimbs_adv
+	name = "All robolimb blueprints"
+	contains = list(
+	/obj/item/weapon/disk/limb/bishop,
+	/obj/item/weapon/disk/limb/hesphiastos,
+	/obj/item/weapon/disk/limb/morpheus,
+	/obj/item/weapon/disk/limb/veymed,
+	/obj/item/weapon/disk/limb/wardtakahashi,
+	/obj/item/weapon/disk/limb/xion,
+	/obj/item/weapon/disk/limb/zenghu,
+			)
+	cost = 40
+	containertype = /obj/structure/closet/crate/secure/gear
+	containername = "Robolimb blueprints (adv)"
+	access = access_robotics
+
+/datum/supply_packs/robotics/mecha_ripley
+	name = "Circuit Crate (\"Ripley\" APLU)"
+	contains = list(
+			/obj/item/weapon/book/manual/ripley_build_and_repair,
+			/obj/item/weapon/circuitboard/mecha/ripley/main,
+			/obj/item/weapon/circuitboard/mecha/ripley/peripherals
+			)
+	cost = 30
+	containertype = /obj/structure/closet/crate/secure
+	containername = "APLU \"Ripley\" Circuit Crate"
+	access = access_robotics
+
+/datum/supply_packs/robotics/mecha_odysseus
```

Comment:
Well if that's the case, who is going to build the parts to the mech if you don't have anyone to build the circuits? Robotics has a circuit printer, right?

I assume you keep Research and Robotics separated, so if it's research levels you're worried about, I'd just suggest lowering the research requirements for the APLU and Ody if you want the circuits to still be accessible. It takes very little research to make them, though I can see how this would still be a blocker if there simply _is no scientist_ to research them.